### PR TITLE
Onboard Sandboxed Containers Operator

### PR DESCRIPTION
**Pull request by wainersm**
Getting the [Sandboxed Containers Operator](https://github.com/openshift/sandboxed-containers-operator) onboarded.

### CODE REVIEW

File: ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- from: ocp_builder_golang-1.13
+  to: sandboxed-containers-operator
+operator:
+  bundles:
+  - as: sandboxed-containers-operator-bundle
+    dockerfile_path: bundle.Dockerfile
```
[the remark this answers is not in the document]
Comment (wainersm) [reply ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-master.yaml:21]: Hi @petr-muller , I will work with @jensfr to fix this. Thanks for the comment!

**Review by petr-muller — APPROVED** [review]

**Pull request merged**